package.json

```
@@ -148,24 +147,23 @@
     "punycode": "^2.1.1",
     "qrcode-generator": "1.4.1",
     "ramda": "^0.24.1",
-    "react": "^15.6.2",
+    "react": "^16.0.0",
```

We could put a newer version here to make sure we don't ever see a downgrade:

```suggestion
    "react": "^16.12.0",
```

Yarn resolved this dependency to `16.12.0` so this seems safe enough.